backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/badge/BpkBadge.kt

```diff
@@ -57,18 +56,36 @@ fun BpkBadge(
   type: BpkBadgeType,
   modifier: Modifier = Modifier,
   icon: BpkIcon? = null,
+) {
+  BpkBadge(
+    text = text,
+    backgroundColor = type.backgroundColor,
+    contentColor = type.contentColor,
+    borderColor = type.borderColor,
+    modifier = modifier,
+    icon = icon,
+  )
+}
+
+@Composable
+internal fun BpkBadge(
```

Comment:
there's a new branded badge type that we can add instead of this and kill 2 birds with 1 stone